src/Middleware/CORS/src/PublicAPI.Shipped.txt

```diff
@@ -13,8 +13,6 @@
 ~Microsoft.AspNetCore.Cors.Infrastructure.CorsOptions.GetPolicy(string name) -> Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicy
 ~Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicy.ExposedHeaders.get -> System.Collections.Generic.IList<string>
 ~Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicy.Headers.get -> System.Collections.Generic.IList<string>
-~Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicyBuilder.AllowAnyHeader() -> Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicyBuilder
```

Comment:
Can you please revert the changes to the Shipped file?  It gets updated automatically when we release.

Reply:
i reverted this and other wrong commits, now build pipelines run successfully

Reply:
Thanks!